### PR TITLE
Fix create_tidb_cluster_release variable does not work

### LINKED ISSUE #1057: `create_tidb_cluster_release` variable in aws terraform script does not work
## Bug Report

```
[root@172.16.4.248 tidb-operator]$grep -r create_tidb_cluster_release *
deploy/modules/aws/tidb-cluster/variables.tf:variable "create_tidb_cluster_release" {
[root@172.16.4.248 tidb-operator]$
```

This variable is not used anywhere.

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

### What problem does this PR solve? 
close #1057 

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:
 
 ```release-note
Fix that the create_tidb_cluster_release variable in AWS terraform script does not work.
 ```

@DanielZhangQD @tennix @cofyc PTAL

### CODE REVIEW

File: deploy/modules/share/tidb-cluster-release/main.tf [hunk 1]
```diff
@@ -22,6 +23,7 @@ data "helm_repository" "pingcap" {
 }
 
 resource "helm_release" "tidb-cluster" {
+  count = var.create ? 1 : 0
```
Comment (DanielZhangQD) [inline deploy/modules/share/tidb-cluster-release/main.tf]: The '=' should be aligned with others in the same block.
Same comments for the other updates in this file.

Comment (tennix) [reply deploy/modules/share/tidb-cluster-release/main.tf]: I think we need to add Terraform `fmt` and `validate` to CI.

Comment (tennix) [reply deploy/modules/share/tidb-cluster-release/main.tf]: There is already an issue tracking this https://github.com/pingcap/tidb-operator/issues/823

Comment (aylei) [reply deploy/modules/share/tidb-cluster-release/main.tf]: if there's a blank line between, then the `=` don't have to be aligned

**Pull request merged**